cpp/ps0001/cpp_ps0001_soln.cpp

```diff
@@ -0,0 +1,29 @@
+ #include <iostream>
+using namespace std;
+int main()
+{
+ // promptiung user for 3 numbers to add 1/2
+ unsigned short num1;
+ unsigned short num2;
+ unsigned short num3;
+ unsigned short result;
+
+ cout << "please enter the first number \n";
+ cin >> num1;
+ 
+ cout << "please enter the second number \n";
+ cin >> num2; 
+ 
+ cout << "please enter the third number \n";
+ cin >> num3;
+

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add